### PR TITLE
App Level Templating Inheritance

### PR DESCRIPTION
**Pull request by jasperboyd**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/5576)

### CODE REVIEW

File: integrationServices/slatwallcms/BaseApplication.cfc [hunk 1]
```diff
@@ -191,16 +192,7 @@ Notes:
 			if(!isnull(entityTemplateContent)){
 				arguments.slatwallScope.setContent( entityTemplateContent );
 				var contentTemplateFile = entityTemplateContent.setting('contentTemplateFile',[entityTemplateContent]);
-				if(!isNull(contentTemplateFile)){
```
Comment (rmarchandTen24) [inline integrationServices/slatwallcms/BaseApplication.cfc]: @jasperboyd not sure why we would remove this code. Doesn't it render 404 if the setting is null?

Comment (jasperboyd) [inline integrationServices/slatwallcms/BaseApplication.cfc]: @rmarchandTen24 I just moved it down to line 218

**Comment by rmarchandTen24** [comment]
@jasperboyd circle ci is saying that the deploy site test didn't deploy the tags template folder. Can you look into this?

**Comment by jasperboyd** [comment]
@rmarchandTen24 I have appeased the mighty CI ;)

**Pull request merged**